src/main/java/shef/servlets/UserServlet.java

```diff
@@ -112,7 +112,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     } catch (EntityNotFoundException e) {
       // Create a new User entity with data from the request.
       user = new Entity(userKey);
-      redirectUrl = "/account-creation-finish.html";
+      redirectUrl = "/quiz.html";
```

Comment:
Do you also allow users to skip this?

Reply:
Yes, the user can choose to not check any of the boxes, that is an acceptable response.
That just means the user doesn't have any specific preferences.

Reply:
However, there was actually a bug where if you selected no preferences, the program would crash. I noticed it and fixed it thanks to your comment. Now, it just sets your preferences as empty.